arith_op_float.c

```diff
@@ -3,8 +3,10 @@
 int main()
 {
     float num1, num2, res;
-    printf("Enter any two number: ");
-    scanf("%f%f", &num1, &num2);
+    printf("Enter First Number: ");
```

Comment:
Use \n in printf() function. It simply increases the readability.

Reply:
added \n in the code.